scripts/modules/ui.js

```diff
@@ -232,14 +294,14 @@ function getStatusWithColor(status, forTable = false) {
 	}
 
 	const statusConfig = {
-		done: { color: chalk.green, icon: '✅', tableIcon: '✓' },
-		completed: { color: chalk.green, icon: '✅', tableIcon: '✓' },
-		pending: { color: chalk.yellow, icon: '⏱️', tableIcon: '⏱' },
+		done: { color: chalk.green, icon: '✓', tableIcon: '✓' },
+		completed: { color: chalk.green, icon: '✓', tableIcon: '✓' },
+		pending: { color: chalk.yellow, icon: '○', tableIcon: '⏱' },
 		'in-progress': { color: chalk.hex('#FFA500'), icon: '🔄', tableIcon: '►' },
-		deferred: { color: chalk.gray, icon: '⏱️', tableIcon: '⏱' },
-		blocked: { color: chalk.red, icon: '❌', tableIcon: '✗' },
-		review: { color: chalk.magenta, icon: '👀', tableIcon: '👁' },
-		cancelled: { color: chalk.gray, icon: '❌', tableIcon: '✗' }
+		deferred: { color: chalk.gray, icon: 'x', tableIcon: '⏱' },
+		blocked: { color: chalk.red, icon: '!', tableIcon: '✗' },
+		review: { color: chalk.magenta, icon: '?', tableIcon: '?' },
+		cancelled: { color: chalk.gray, icon: '❌', tableIcon: 'x' }
```

Comment:
wouldn't you want to change this to x ?
```suggestion
		cancelled: { color: chalk.gray, icon: 'x', tableIcon: 'x' }
```